evap/results/templates/results_index.html

```diff
@@ -77,7 +77,7 @@ <h5 class="card-title mb-lg-0">
                 <div data-col="name" class="col-order">{% trans 'Evaluation' %}</div>
                 <div data-col="semester" class="col-order">{% trans 'Semester' %}</div>
                 <div data-col="responsible" class="col-order">{% trans 'Responsible' %}</div>
-                <div data-col="participants">{% trans 'Voters' %}</div>
+                <div data-col="participants" class="col-order">{% trans 'Voters' %}</div>
```

Comment:
Is there any reason we intermingle `participants` and `voters` here (@janno42 @niklasmohrin)? :D Seems like this could be cleaned up

Reply:
Good catch! Should be called `voters`

Reply:
offline discussion: -> unify to "voters"